wp_gutenberg_ref_update_prs.sh

```diff
@@ -0,0 +1,283 @@
+# Ensure javascript dependencies are up-to-date
+read -r -p "Run 'npm ci' to ensure javascript dependencies are up-to-date? (y/n) " -n 1 PERFORM_NPM_CI
+echo ""
+confirm_to_proceed "Script execution for creating WPApps PR's will take ~15-20 mins. Proceed? "
+if [[ $PERFORM_NPM_CI =~ ^[Yy]$ ]]; then
+    execute "npm" "ci"
+fi
+
+# Create Git branch
+GUTENBERG_MOBILE_BRANCH="$BRANCH_NAME"
+ohai "Create Git branch '$GUTENBERG_MOBILE_BRANCH' in gutenberg-mobile."
+execute "git" "switch" "-c" "$GUTENBERG_MOBILE_BRANCH"
+
+# Create Git branch in Gutenberg
+GB_BRANCH="rnmobile/$GUTENBERG_MOBILE_BRANCH"
+ohai "Create Git branch '$GB_BRANCH' in gutenberg."
+cd gutenberg
+execute "git" "switch" "-c" "$GB_BRANCH"
+cd ..
+
+# Make sure podfile is updated
+ohai "Make sure podfile is updated"
+PRE_IOS_COMMAND="npm run core preios"
+eval "$PRE_IOS_COMMAND"
+
+# If preios results in changes, commit them
+cd gutenberg
+if [[ -n "$(git status --porcelain)" ]]; then
+    ohai "Commit changes from '$PRE_IOS_COMMAND'"
+    execute "git" "commit" "-a" "-m" "Release script: Update with changes from '$PRE_IOS_COMMAND'"
+else
+    ohai "There were no changes from '$PRE_IOS_COMMAND' to be committed."
+fi
+cd ..
+
+# Commit updates to gutenberg submodule
+if [[ -n "$(git status --porcelain)" ]]; then
+    ohai "Commit updates to gutenberg submodule"
+    execute "git" "add" "gutenberg"
+    execute "git" "commit" "-m" "Release script: Update gutenberg ref"
+else
+    ohai "There were no changes from submodule update to be committed."
+fi
+
+# Update the bundles
+ohai "Update the bundles"
+npm run bundle || abort "Error: 'npm bundle' failed.\nIf there is an error stating something like \"Command 'bundle' unrecognized.\" above, perhaps try running 'rm -rf node_modules gutenberg/node_modules && npm ci'."
+
+# Commit bundle changes
+if [[ -n "$(git status --porcelain)" ]]; then
+    ohai "Commit bundle changes"
+    execute "git" "add" "bundle/"
+    execute "git" "commit" "-m" "Release script: Update bundle for: $BRANCH_NAME"
+else
+    ohai "There were no changes from bundle update to be committed."
+fi
+
+
+#####
+# Create PRs
+#####
+
+# Replace version number in GB-Mobile PR template
+PR_BODY="## Description
+This PR is for publishing the $GUTENBERG_MOBILE_BRANCH branch of gutenberg-mobile to S3 for testing in WPApps repos.
+
+
+
+PR Submission Checklist
+- [ ] Update target branch away from trunk unless this is a release PR
+- [ ] I have considered if this change warrants user-facing release notes and have added them to \`RELEASE-NOTES.txt\` if necessary."
+execute "git" "push" "-u" "origin" "HEAD"
+
+# Create Draft GB-Mobile Release PR in GitHub
+GB_MOBILE_PR_URL=$(execute "gh" "pr" "create" \
+"--title" "App Integration for $GUTENBERG_MOBILE_BRANCH" \
+"--body" "$PR_BODY" \
+"--repo" "$MOBILE_REPO/gutenberg-mobile" \
+"--head" "$MOBILE_REPO:$GUTENBERG_MOBILE_BRANCH" \
+"--base" "$GUTENBERG_MOBILE_TARGET_BRANCH" \
+"--label" "$GUTENBERG_MOBILE_PR_LABEL" \
+"--draft")
```

Comment:
Noting for myself for later that gb-mobile bridge publish to S3 job only fires if PR is not a draft. Thinking maybe we can  remove -draft flag and add a "DO NOT MERGE" label for gutenberg-mobile PR.

Reply:
Good observation. I noticed it as well. I had made a PR ready for review just to get it to build even though I wanted it to remain as a draft.